modules/vaos/app/models/vaos/appointment_request_form.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+
+require 'active_model'
+require 'common/models/form'
+
+module VAOS
+  class AppointmentRequestForm < Common::Form
```

Comment:
Please add top level class [rubydocs](https://www.rubydoc.info/github/department-of-veterans-affairs/vets-api and public method docs

Reply:
Will do, in follow-up PR. Will open a ticket before merging to document that debt.

Reply:
department-of-veterans-affairs/va.gov-team#3689